### PR TITLE
Merge to CI 3.1.13 and add docker files

### PR DESCRIPTION
**Pull request by hadrylouis**
Changes :

- Use Docker
- Migrate to PHP 8.1 (CI 3 is not compatible yet with PHP >=8.2)
- Migrate to CodeIgniter 3.1.13
- Use nginx instead of apache

### CODE REVIEW

**Comment by richmartins** [comment]
LGTM

**Comment by hadrylouis** [comment]
@richmartins New changes have been made, could you please review the PR again and then merge it if everything seems okay to you.

FYI i switched from mysql to mariadb

File: src/application/controllers/Auth.php
Comment (richmartins) [inline src/application/controllers/Auth.php:1]: we should add this as a env variable at somepoint, but for now it's fine!

**Comment by richmartins** [comment]
Everything seems good. Merging RN! thanks

**Pull request merged**